### ISSUE TITLE
Weights file missing for PascalVOC training?

### ISSUE THREAD

**Issue by ibadiu** [choasup/caffe-yolo9000#4]
Hi, 
First of all, thanks for publishing your implementation of YOLO9000!
Following up a previous question about the missing caffemodel:
WEIGHTS=./gnet_yolo_region_darknet_v3_pretrain_iter_600000.caffemodel
Could you make it available even it the training was not optimal?

Otherwise, could you specify which caffe configuration files (solver and prototxt) you used for training it?

Thanks

**Comment by choasUp** [comment]
configuration files in 'examples/darknet_v3/'.
pretrain model has changed to be 'gnet_yolo_region_darknet_v3_pretrain_iter_360000.caffemodel',
url: https://pan.baidu.com/s/1o7VcHui 
password: 5qpy

**Comment by ibadiu** [comment]
Thanks for your reply!
One more question concerning the configuration files. Shouldn't the architecture of the classification network be changed to reflect the changes from Table 6 of the YOLO9000 paper (https://arxiv.org/pdf/1612.08242.pdf)?
More precisely, the region related layers (conv_reg and det_loss) be replaced with a convolution, an average pooling and a softmax when pre-training with imagenet?

**Comment by choasUp** [comment]
Yes, you are right.I am sorry that I didn't read paper carefully. I figured out why my pre-train model didn't get a good performance, maybe it caused poor mAP. I will try to re-train the pre-train model.

发自我的 iPhone

**Comment by ibadiu** [comment]
Cool, let me know what you obtain once the retraining is done.

**Comment by Junsong-Wang** [comment]
The mAP of VOC2007 is only 0.536825 based on the model provided, have you retrained the model that is much higher accuracy as published in their paper.

**Comment by choasUp** [comment]
I will try to retrain it in this month.
